README.md

```diff
@@ -1,24 +1,70 @@
 
-![Inventory management UI](https://cloud.githubusercontent.com/assets/5801549/15571240/01a76534-233a-11e6-8cd1-1281c7de22f9.png)
+## Overview
 
-# Documentation
-User guide: <a href="https://virtocommerce.com/docs/vc2userguide/fulfillment" target="_blank">Fulfillment</a>
+Inventory management is a system of stock level controlling and fullfilment centers management. 
 
+Inventory is often the largest item a business has in its current assets, meaning it must be accurately monitored. Inventory is counted and valued at the end of each accounting period to determine the company's profits or losses.
+
+In the ecommerce sector, a day-to-day management of inventory helps increase business intelligence and visibility. Ecommerce business owners have better ongoing control when they have constant eyes on inventory.
+
+## VirtoCommenrce Inventory Module Description
+
+VirtoCommerce.Inventory module represents a store inventory management system. The main goal of the Inventory module is stock level controlling and managing of fulfillment centers.
+
+The VC Inventory module is customizable and supports metafields.
+
+The Inventory module content can be both exported and imported using the VC Export/Import functionality.
+
+![Inventory module](docs/media/screen-inventory.png)
+
+Fields description :
+
+1. 'In stock' - total quantity of product items allowed in stock;
+1. 'Reserved'- in VC this field has no logics and is simply a UI element;
+1. 'Reoder min. qty'- the minimum quantity of products that can be reordered;
+1. 'Allow preoder' - if switched on means that the system will allow order products that are still not in stock. By using the calender, the admin can set up the time period for preoder availability;
+1. 'Preorder qty field'- the admin can specify the product quantity that can be preordered;  
+1. 'Allow backorder'- if switched on means that the system will allow order products that are no longer in stock. The admin can set up the time period for backorder availability;
+1. 'Backorder qty' field- the admin can specify the product quantity that can be backordered.
+
+Some products may not require inventory tracking. For example, services, second hand or custom-made goods. In this case, a store owner can select not to track inventory, by switching off the 'Track Inventory' option on the Physical item details blade
+
+![Track Inventory](docs/media/screen-track-inventory.png)
+
+### Fulfillment Centers
+
+VC fulfillment includes the steps involved in receiving, processing and delivering orders to end customers. Usually the process of fulfillment in ecommerce inlcudes receiving inventory from a supplier (including its counting, instecting, labeling and etc.), storage of the inventory, order processing (including preparing, packing and exchange if necessary) and shipping. There also may be returns that are counted according to the store rules and conditions. In Virto Commerce manager fulfillment lifecycle looks as follows:
```

Comment:
Typo
Usually **,** the process of fulfillment in ecommerce includes receiving inventory from a supplier (including its counting, **inspecting,** labeling and etc.), storage of the inventory, order processing (including preparing, packing and exchange if necessary) and shipping.